data/json/materials.json

```diff
@@ -159,7 +159,7 @@
         ],
         "acid_resist" : 0,
         "elec_resist" : 0,
-        "fire_resist" : 0,
+        "fire_resist" : 1,
```

Comment:
Water will evaporate but I've **never** seen it catch fire.  (Lighting one's tap is a way to prove you've got gas contamination in the water line.)